docs/book/src/SUMMARY.md

```diff
@@ -108,3 +108,4 @@
     - [Code Review in Cluster API](./REVIEWING.md)
     - [Version Support](./reference/versions.md)
     - [Roadmap](./roadmap.md)
+    - [Supported Labels/Annotations](./reference/labels_and_annotations.md)
```

Comment:
Could we also add a link to this in the API reference? i.e. `./reference/api_reference.md`

Reply:
Did it.